android_jni/avifandroidjni/src/main/jni/libavif_jni.cc

```diff
@@ -74,6 +75,20 @@ bool CreateDecoderAndParse(AvifDecoderWrapper* const decoder,
     LOGE("Failed to parse AVIF image: %s.", avifResultToString(res));
     return false;
   }
+
+  avifDiagnostics diag;
+  // If the image does not have a 'clap' property or if it is invalid, then we
```

Comment:
Optional: The "the image does not have a 'clap' property" part can be omitted. What needs to be documented is our policy on invalid clap properties.

Reply:
I've rewritten so that it covers both the cases. "If the image does not have a valid 'clap' property ....".